plugin/data_masking/src/udf/udf_gen_dictionary_load.cc

```diff
@@ -111,6 +111,7 @@ static std::string _gen_dictionary_load(const char *dictionary_path,
         if (s.length() > 0) list.push_back(s);
       }
       if (list.size() > 0) {
+        std::sort(list.begin(), list.end());
```

Comment:
Can we make this list a `std::unordered_set<std::string>`?

Reply:
No, there is a function that selects a randomly chosen item from a dictionary. We could make it an unordered_set and a vector, but that would increase the memory usage a bit...